build/packages-template/bbb-graphql-middleware/before-install.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash -e
+
+case "$1" in
+  install|upgrade|1|2)
+
+    # the config is not longer env vars at /etc/bigbluebutton/bbb-graphql-middleware
+    if [ -f /etc/bigbluebutton/bbb-graphql-middleware ]; then
```

Comment:
I thought you'd be removing the old `/etc/default/bbb-graphql-middleware*`
I don't think we should be touching `/etc/bigbluebutton` at all. It's reserved for the admins.

Reply:
I copied this part from your PR 🤣
you are right, we should remove `/etc/default/bbb-graphql-middleware`!